packs/hpe-icsp/actions/icsp_ca_get.yaml

```diff
@@ -0,0 +1,13 @@
+---
+  name: "icsp_ca_get"
+  runner_type: "run-python"
+  description: "Retreive CA Certificate from ICSP"
```

Comment:
Out of curiosity since it's not really immediately clear from the code - what does this action return (an object or something else)?